### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by LandonSchropp**
* Bump Ruby to 2.6.5
* Bump dependencies
* Fix the Rubocop configuration
* Add rules from tesla-site
* Clear out TODOs on cops

### CODE REVIEW

**Review by optimumtk — APPROVED** [review]

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.6.4
+2.6.5
```
Comment (jorge0136) [inline .ruby-version:1]: Does this imply that we should match this version in our projects too? Perhaps this is just the runtime for this project and the others are ok to be on `2.6.3`? AKA here is [site](https://github.com/optimumenergyco/tesla-site/blob/master/.ruby-version).

**Review by jorge0136 — COMMENTED** [review]
One question, nothing blocking.

**Review by BrockBeaudry — APPROVED** [review]
LGTM aside from the build failure

File: .ruby-version (hunk 1 above)
Comment (LandonSchropp) [reply .ruby-version:1]: Nope. Gems tend to be very flexible in the versions of Ruby that they support. Usually, authors of big gems try to stick to core language features. In our case, all of the Ruby code is in the Rubocop gem itself, so it shouldn't matter.

File: optimum_energy_rubocop.yml [hunk 2]
```diff
@@ -73,6 +88,39 @@ Layout/SpaceBeforeFirstArg:
 Layout/SpaceInsideStringInterpolation:
   EnforcedStyle: space
 
+###########
+# METRICS #
+###########
+
+Metrics/AbcSize:
+  Enabled: false
```
Comment (optijon) [inline optimum_energy_rubocop.yml:96]: This seems like a useful rule. Why as it disabled?

File: optimum_energy_rubocop.yml [hunk 3]
```diff
@@ -73,6 +88,39 @@ Layout/SpaceBeforeFirstArg:
...
+  Max: 100
+  Exclude:
+    - 'spec/**/*'
+
+Metrics/ClassLength:
+  Enabled: true
+  Max: 300
+
+Metrics/CyclomaticComplexity:
+  Enabled: false
```
Comment (optijon) [inline optimum_energy_rubocop.yml:109]: This one seems useful too. Why was it disabled?

Comment (LandonSchropp) [reply optimum_energy_rubocop.yml:109]: The problem is it's not discerning. It doesn't handle DSLs well, and there are rare exceptions when we want longer methods. I think it's better to call this out in PR reviews.

File: optimum_energy_rubocop.yml (hunk 2 above)
Comment (LandonSchropp) [reply optimum_energy_rubocop.yml:96]: Ditto to the answer before.

**Review by jorge0136 — APPROVED** [review]

**Review by optijon — APPROVED** [review]

**Pull request merged**